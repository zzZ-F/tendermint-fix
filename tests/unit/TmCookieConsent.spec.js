import { shallowMount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmCookieConsent } from "@/index"

describe("TmCookieConsent.vue", () => {
  let wrapper
  beforeEach(async () => {
    const value = "button"
    wrapper = shallowMount(TmCookieConsent, {
      propsData: { value }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
