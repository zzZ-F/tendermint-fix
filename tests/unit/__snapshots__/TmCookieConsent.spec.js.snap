// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmCookieConsent.vue has the expected html structure 1`] = `<div class="tm-cookie-consent" value="button"></div>`;
